README.md

```diff
@@ -1,65 +1,21 @@
-Bowling Challenge in Ruby
-=================
+# Bowling Challenge - Ruby
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+Our weekend challenge for the fifth week of the Makers Academy boot camp was to write a program that calculates the score of a bowling game, with a view to trying the same again in JavaScript. We were not tasked with fleshing it out into a playable game with a user interface, so I kept it very streamlined.
 
-## The Task
+## My Approach
```

Comment:
Very well documented approach. You clearly explained how you planned to implement the software construction, the challenges you faced and how you solved them and the improvements you think can be applied to the code.